libr/anal/p/anal_arm_cs.c

```diff
@@ -4439,98 +4440,190 @@ static void op_fillval(RAnal *anal, RAnalOp *op, csh handle, cs_insn *insn, int
 	}
 }
 
+static R_TH_LOCAL csh cs_handle = 0;
```

Comment:
There's a PR from @iotaaxel moving those variables into RAnal. will be merged after 5.7.8, so we can make capstone plugins a bit more threadsafe

Reply:
sure, but I want to merge this tonight